apps/forecaster/src/app/api/humidity/route.ts

```diff
@@ -18,3 +18,5 @@ export async function GET(request: NextRequest) {
     console.error(error);
   }
 }
+// Don't use vercel caching...
+export const fetchCache = 'force-no-store';
```

Comment:
Interestingly setting this value not only disables vercel's Data Cache but also allow logs to appear 🤷🏽